test/objectwrap.js

```diff
@@ -61,6 +66,9 @@ async function test (binding) {
 
       obj.testGetSetT = 'instance getset 4';
       assert.strictEqual(obj.testGetSetT, 'instance getset 4');
+
+      try { clazz.prototype.testGetSet = 'instance getset'; } catch (e) { }
+      try { clazz.prototype.testGetSetT = 'instance getset'; } catch (e) { }
```

Comment:
```suggestion
      assert.throws(() => clazz.prototype.testGetSet, /Invalid argument/);
      assert.throws(() => clazz.prototype.testGetSetT, /Invalid argument/);
```

Reply:
Not sure about this one. Don't you want to test the setter? `prototype.testGetSet = 'instance get set'` does actually call the setter with `prototype` as the `this` instead of the constructed object.

```js
class Abc { set v(v) { console.log(this instanceof Abc); } }
const abc = new Abc();
abc.v = 2;
// log: true
Abc.prototype.v = 2;
// log: false
```

Reply:
The suggestion here might be:

```suggestion
      assert.throws(() => { clazz.prototype.testGetSet = 'instance getset'; }, /Invalid argument/);
      assert.throws(() => { clazz.prototype.testGetSetT = 'instance getset'; }, /Invalid argument/);
```

Reply:
@legendecas @chearon yes, that's what I meant. The important thing is that we use `assert.throws()` and that we check for the right exception with the regex.